### DISCUSSION TITLE
Omni-key not working

### DISCUSSION THREAD

**Discussion by AnY-0nE21** [Lailloken/Exile-UI#436] [Q&A] [answered]
Hi, I have a problem with omni-key. I came across it while using map-info, I have tried to use omni key to run map info module but the only thing that happened is my screen flashing for 2s and thats it.

I have tried to look for answers in discussion tab and the only one I found was sth related to skill tree calibration taht was about changing DirectX i tried DirectX11 (flashing thing) DirectX 12 (nothing happened) and vulcan (nothing happened). 
I also tried enabling and diabling HDR in windows settings but still nothing works, so I came here for help

**Comment by Lailloken** [comment, marked as answer]
Switching to DX12 is the correct thing to do. If you're already running v1.54.6 (released today), then the `inventory` pixel-check is mandatory for item-related features. So make sure it's calibrated correctly in the `screen-checks` section of the settings.

**Reply by AnY-0nE21** [reply]
both gamescreen and inventory are calibrated and tested positive but still nothing happens with map

**Reply by AnY-0nE21** [reply]
Oh nvm it just worked
sorry for trouble and thanks for help :)
